recipes-domd/agl/files/meta-agl-bsp/conf/include/agl_rcar_xt.inc

```diff
@@ -41,7 +41,6 @@ IMAGE_INSTALL_append_rcar-gen3 = " \
 	kernel-module-pvrsrvkm \
 	kernel-module-vsp2 \
 	kernel-module-vspm-if \
-	omx-user-module \
```

Comment:
Is this a part of adding vGPU support to AGL?

Reply:
will fix

Reply:
WARNING: /media/build-workspace/ces2018/build/tmp/work/x86_64-xt-linux/domd-agl-image-weston/1.0-r0/repo/meta-renesas-rcar-gen3/meta-rcar-gen3/recipes-multimedia/omx-module/omx-user-module.bb: Unable to get checksum for omx-user-module SRC_URI entry EVARTM0AC0000XCMCTL30SL40C.tar.bz2: file could not be found

cannot be built from sources